tests/lang/s01_basics/s06_data_types/t7_strings.nim

```diff
@@ -37,7 +37,7 @@ block:
   doAssert pltLen == 9,        "string length excludes last null byte"
   doAssert pltLen == vmLen,    "VM and runtime length are the same"
 
-block:
+block stringsValid_3:
```

Comment:
The same applies to other string construction examples - most likely these blocks will be turned into manual heading later, or otherwise would be visible to the specification reader.